tool/debian/build.sh

```diff
@@ -86,26 +86,22 @@ if [ -n "$BUILD_ID" ]; then
 fi
 
 FOLDER=$FOLDER_PREFIX-cpp
-if [ $CUDA = "ON" ]; then
-  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
-fi
-
 if [ $PYTHON = "ON" ]
 then
   FOLDER=$FOLDER/python-singa
```

Comment:
These directories are only a temporary solution for saving the control files. We do not need them to be perfect. Later if we use CPack, these directories could be removed as the control files will be generated automatically.